openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js

```diff
@@ -32,10 +31,7 @@ const expirationTime = {
 };
 
 const createExpiryDate = (expiry, days) => {
-  const currentDate = Date.now();
-  const endDate = moment(currentDate, 'x')
-    .add(expiry, days)
-    .format(`ddd Do MMMM, YYYY,hh:mm A.`);
+  const endDate = getExpiryDateTimeFromDate(expiry, days);
```

Comment:
We don't need this function anymore `createExpiryDate`

Reply:
Done.